src/DocumentationBrowserViewExtension/PackageDocumentationManager.cs

```diff
@@ -39,17 +44,58 @@ public static PackageDocumentationManager Instance
             }
         }
 
+        public void AddDynamoPaths(IPathManager pathManager)
+        {
+            if (pathManager is null)
+            {
+                return;
+            }
+
+            if (!string.IsNullOrEmpty(pathManager.DynamoCoreDirectory))
+            {
+                var coreDir = new DirectoryInfo(Path.Combine(pathManager.DynamoCoreDirectory, FALLBACK_DOC_DIRECTORY_NAME));
+                dynamoCoreFallbackHostPath = coreDir.Exists ? coreDir : null;
+            }
+
+            if (!string.IsNullOrEmpty(pathManager.HostApplicationDirectory))
+            {
+                var hostDir = new DirectoryInfo(Path.Combine(pathManager.HostApplicationDirectory, FALLBACK_DOC_DIRECTORY_NAME));
+                hostDynamoFallbackDocPath = hostDir.Exists ? hostDir : null;
+            }   
+        }
+
         private PackageDocumentationManager() { }
 
         /// <summary>
         /// Retrieves the markdown node documentation file associated with the input node namespace
         /// </summary>
         /// <param name="nodeNamespace">Namespace of the node to lookup documentation for</param>
         /// <returns></returns>
-        public string GetAnnotationDoc(string nodeNamespace)
+        public string GetAnnotationDoc(string nodeNamespace, string packageName)
```

Comment:
hmm I'm not sure what to do about this:
On one hand, this is a public method, on the other this is not `dynamocCore` - it's a method in use inside this particular extension.

Why is this public anyway?

@QilongTang @aparajit-pratap what do you think?

Reply:
I think last time we discussed extension was not part of our API scope? Although clarification of this was not part of 2.12.0 scope. We originally plan to clarify that Dynamo View layer is not. We could add that if we want to break the rule here.